.github/workflows/pythonpublish.yml

```diff
@@ -13,11 +13,10 @@ jobs:
         python-version: '3.x'
     - name: Install dependencies
       run: |
-        pip install setuptools wheel
+        pip install setuptools wheel build
     - name: Build distribution 📦
       run: |
-        pip install .
-        python setup.py sdist bdist_wheel
+        python -m build
```

Comment:
Did you test this already? e.g. on a testpypi?

Reply:
sdist generation was one of the tests I had to resolve, and IIRC it worked; I'll test locally properly with nektos/act.

Reply:
Thanks for making me upload to TestPyPI as it had identified an issue with the `pyproject.toml` with the project URLs. I have uploaded it successfully. However, I have identified another problem, more in the other review thread.